gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -174,7 +178,7 @@ private void ensureLockfileIsUpToDate(List<ProductDependency> productDeps) {
                         "%s does not exist, please run `./gradlew %s --write-locks` and commit the resultant file",
                         relativePath, getName()));
             } else {
-                String fromDisk = GFileUtils.readFile(lockfile);
+                String fromDisk = new String(Files.readAllBytes(lockfile.toPath()), StandardCharsets.UTF_8);
```

Comment:
If we are using java 11 in this repo yet, we can do `Files.readString`, which also helpfully uses UTF-8 by default.

Reply:
Updated